src/init.cpp

```diff
@@ -1662,6 +1662,18 @@ bool AppInit2(boost::thread_group &threadGroup, CScheduler &scheduler) {
                     break;
                 }
 
+                if (!fReindex) {
+                    CBlockIndex *tip = chainActive.Tip();
+                    if (tip && tip->nHeight >= chainparams.GetConsensus().nSigmaStartBlock) {
+                        const uint256* phash = tip->phashBlock;
+                        if (pblocktree->GetBlockIndexVersion(*phash) < CLIENT_VERSION) {
```

Comment:
We need to compare it with the first version having sigma (0.13.8+ or whatever it would be), not the current version